plugin/switch.vim

```diff
@@ -216,6 +216,18 @@ autocmd FileType scala let b:switch_definitions =
       \ [
       \   g:switch_builtins.scala_string,
       \ ]
+autocmd FileType gitrebase let b:switch_definitions =
+      \ [
+      \   [ 'pick', 'fixup', 'reword', 'edit', 'squash', 'exec', 'drop' ],
+      \   { '\^p ', 'fixup' },
+      \   { '\^f ', 'reword ' },
+      \   { '\^r ', 'edit' },
+      \   { '\^e ', 'squash ' },
+      \   { '\^s ', 'exec ' },
+      \   { '\^x ', 'drop ' },
+      \   { '\^d ', 'pick ' },
```

Comment:
And it should also be `{ '^d': 'pick ' }`, with a `:`, not a `,`. I'm not sure what this syntax ends up being parsed as.